new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiMultiModel.java

```diff
@@ -0,0 +1,332 @@
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.createDockerRegistrySecret;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.createDomainAndVerify;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.createSecretWithUsernamePassword;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.installAndVerifyOperator;
+import static org.awaitility.Awaitility.with;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@DisplayName("Test to create model-in-image domain with a ConfigMap that contains multiple WDT models")
+@IntegrationTest
+class ItMiiMultiModel implements LoggedTest {
+
+  private static String domainNamespace = null;
+  private static ConditionFactory withStandardRetryPolicy = null;
+  
+  private static String domainUid = "mii-multimodel-cm-domain";
+
+  private static String adminServerPodName = String.format("%s-%s", domainUid, ADMIN_SERVER_NAME_BASE);
+  private static String managedServerPrefix = String.format("%s-%s", domainUid, MANAGED_SERVER_NAME_BASE);
+  private static int replicaCount = 2;
+
+  /**
+   * Perform initialization for all the tests in this class.
+   * Set up the necessary namespaces and install the operator in the first namespace.
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+    // create standard, reusable retry/backoff policy
+    withStandardRetryPolicy = with().pollDelay(2, SECONDS)
+        .and().with().pollInterval(10, SECONDS)
+        .atMost(5, MINUTES).await();
+
+    // get a new unique opNamespace
+    logger.info("Creating unique namespace for Operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    String opNamespace = namespaces.get(0);
+
+    logger.info("Creating unique namespace for Domain");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domainNamespace = namespaces.get(1);
+
+    // install the operator
+    logger.info("Install an operator in namespace {0}, managing namespace {1}",
+        opNamespace, domainNamespace);
+    installAndVerifyOperator(opNamespace, domainNamespace);
+  }
+
+  /**
+   * Test that two WDT model files in a domain resource's ConfigMap are applied in the expected order.
+   * Create a WebLogic domain with a Kubernetes ConfigMap that contains two WDT model files.
+   * Verify that the effective configuration of the domain is as expected.
+   */
+  @Test
+  @DisplayName("Create model-in-image domain with a ConfigMap that contains multiple model files")
+  @Slow
+  @MustNotRunInParallel
```

Comment:
Why this test shouldn't be run in parallel?

Reply:
removed.